api/lib/infrastructure/serializers/csv/csv-serializer.js

```diff
@@ -31,8 +31,9 @@ function deserializeForSessionsImport(parsedCsvData) {
 
   _verifyHeaders({ csvLineKeys, headers, parsedCsvLine: parsedCsvData[0] });
 
-  parsedCsvData.forEach((line) => {
-    const data = _getDataFromColumnNames({ csvLineKeys, headers, line });
+  parsedCsvData.forEach((lineDTO, index) => {
+    const dataFromColumnName = _getDataFromColumnNames({ csvLineKeys, headers, line: lineDTO });
+    const data = { ...dataFromColumnName, line: index + 1 };
```

Comment:
```suggestion
const data = { ...dataFromColumnName, line: index + 2 };
```

Il ne faut pas prendre en compte la première ligne qui est celle des headers